README.md

```diff
@@ -6,6 +6,14 @@
 
 # PGMax
 
+## PGMax is now part of DeepMind 🎉!
```

Comment:
Better to make this top-level, above everything else, instead of part of the description

Reply:
Also you should make this notice bigger and stand out more. https://github.com/ylabbe/cosypose/blob/master/README.md Here is an example that I recently came across. You could adapt their html code.

Reply:
these points should be adressed in the new commit